IFC4x3/Properties/l/LongitudinalSpacing_2Fqc$pMNj5C8jRJPnj8nSC/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LongitudinalSpacing_2Fqc$pMNj5C8jRJPnj8nSC" Name="LongitudinalSpacing" UniqueId="8fd26ff3-597b-4530-8b5b-4d9c6d23170c" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Are the still ribs not modelled i.e. can be extracted from geometry?

Reply:
The domain experts wanted to keep it.